### ISSUE TITLE
Suggestion: between() should be inclusive of min, max

### ISSUE THREAD

**Issue by bakatz** [mvallim/java-fluent-validator#49]
I believe the most user-friendly implementation would be (below is Kotlin code, may not convert entirely cleanly to java):
```
fun <E, T : Comparable<E>?> betweenInclusive(min: E, max: E): Predicate<T>? =
    PredicateBuilder.from<T>(LogicalPredicate.not(ObjectPredicate.nullValue()))
            .and(ComparablePredicate.lessThanOrEqual(max).and(ComparablePredicate.greaterThanOrEqual(min)))
```

Which relies on *OrEqual functions. 

Otherwise this can be implemented as a separate method from between, or as a parameter (Boolean inclusive)

**Comment by mvallim** [comment]
Hi Ben

Thanks for the suggestion, I added this new predicate `betweenInclusive`.

https://github.com/mvallim/java-fluent-validator/blob/dcc399d19364117d853ded8099b1d69d9ee1d427/src/main/java/br/com/fluentvalidator/predicate/ComparablePredicate.java#L49

https://github.com/mvallim/java-fluent-validator/blob/dcc399d19364117d853ded8099b1d69d9ee1d427/src/main/java/br/com/fluentvalidator/predicate/ComparablePredicate.java#L62

In the next release will be available.

Regards,

**Comment by bakatz** [comment]
Amazing, thanks for the quick response.

**Comment by mvallim** [comment]
Available in release 1.7.6
